fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -131,6 +131,7 @@
     RESOURCE_RETRIEVAL_FAILURE("Unable to retrieve the resource for given resource name: %s. %s"),
     RESOURCE_STORAGE_FAILURE("Unable to store the resource for resource name %s. %s"),
     RESOURCE_DELETION_FAILURE("Unable to delete the resource for resource name %s. %s"),
+    UNKNOWN_EXCEPTION_MOBSTOR("Unknown error during transaction with mobstor . Additional info %s."),
 
```

Comment:
We cannot reference mobstor in the public api.  This needs to move to the private yahoo packages.